app/client.ml

```diff
@@ -182,6 +199,10 @@ let schedule_orb_build_cmd =
   Term.(term_result (const schedule_orb_build $ setup_log $ remote $ nam $ opam_package $ period)),
   Term.info "orb-build"
 
+let reschedule_cmd =
+  Term.(term_result (const reschedule $ setup_log $ remote $ nam $ next $ const None)),
```

Comment:
`const None` <- this should be `period` (but optional, and default to None)!?

Reply:
I pushed a commit that exposes `--period` (with default `None`).